.github/workflows/rust.yml

```diff
@@ -91,3 +91,16 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - uses: EmbarkStudios/cargo-deny-action@v2
+
+  check:
```

Comment:
I think this name is confusing. My proposal would be just `features`.

Reply:
Note that this does actually execute a `cargo check`. The only difference to a plain `cargo check` is that it does so across all feature permutations. Thus the name is consistent with the above `test` which executes a `cargo test`.

With that in mind, what do you think @djc? I don't feel strongly about it. Happy to change to `features`.

Reply:
There are lots of CI jobs that employ (or could employ) `check` so IMO `check` is not distinctive enough to describe the goal of this job, which is to exercise the Cargo features.

Reply:
```suggestion
  features:
```